interval.go

```diff
@@ -36,7 +36,7 @@ type Interval struct {
 func NewInterval(d time.Duration) *Interval {
 	i := Interval{
 		C:  make(chan struct{}, 1),
-		in: make(chan struct{}),
+		in: make(chan struct{}, 1),
```

Comment:
What was the rationale for making this change? LOL, I don't remember?

Reply:
Drilling down on this right now. The channel blocks if the size is 1. Trying to figure out what is placing more than 1 struct on the channel.